src/test/test_transform.ts

```diff
@@ -413,17 +424,7 @@ test('transform returns a context class when given a Context interface', pass, (
       },
     },
   });
-  expect(result.clientContext).to.eql({
-    name: 'ClientContext',
-    attributes: [
-      {
-        name: 'foo',
-        type: 'string',
-        optional: false,
-      },
-    ],
-    methods: [],
-  });
+  t.true(result.globals.clientContext);
```

Comment:
With latest `ava@1.4.0`, consider using `t.assert(result.globals.clientContext === true)` if you want power-assert on error here.